src/components/icon/Icon.constants.ts

```diff
@@ -1,16 +1,19 @@
-import { IconGlyphs } from '@mattermost/compass-icons/build/IconGlyphs';
+import IconGlyphs from '@mattermost/compass-icons/build/IconGlyphs';
 
 import { TIconGlyph, TIconSize } from './Icon.types';
 
 const DEFAULT_ICON_SIZE = 20;
 
-const ICON_SIZES: TIconSize[] = [10, 12, 16, 20, 28, 32, 40, 52, 64, 104];
+const ICON_SIZES: TIconSize[] = [10, 12, 14, 16, 18, 20, 24, 28, 32, 40, 52, 64, 104];
 
 const ICON_FONT_SIZES: Record<TIconSize, number> = {
     10: 12,
     12: 14,
+    14: 16,
```

Comment:
IMO: we should not ... Reason: the design system should follow their own rules. If absolutely not possible otherwise we can still overwrite the size on component level styling, but this should be handled as last resort!